client/src/components/BookingList/Booking.tsx

```diff
@@ -0,0 +1,39 @@
+import { Booking as IBooking } from '../../interface/Booking';
+import { Avatar, Card, Grid, IconButton, Typography, Box } from '@mui/material';
+import SettingsIcon from '@mui/icons-material/Settings';
+import format from 'date-fns/format';
+
+interface PropTypes {
+  booking: IBooking;
+}
+
+const Booking = ({ booking }: PropTypes) => {
+  return (
+    <Card variant="outlined" sx={{ padding: '0px 10px 10px 10px', margin: '10px 0px' }}>
```

Comment:
```suggestion
    <Card variant="outlined" sx={{ padding: '0px 10px 10px 10px', margin: '10px 0px' }}>
      <CardContent>
```
Nit: We can wrap the contents with the `CardContent` component to stay consistent with MUI semantics 😄